config/manifests/patches/spec-descriptors.yaml

```
@@ -0,0 +1,452 @@
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.internodeSSL
+      description: |
+        Internode SSL port to use when leveraging Node Port network configuration.
+      displayName: Internode SSL Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.native
+      description: |
+        Native transport port to use when leveraging Node Port network configuration.
+      displayName: Native Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.nativeSSL
+      description: |
+        Native transport SSL port to use when leveraging Node Port network configuration.
+      displayName: Native SSL Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: storageConfig
+      description: |
+        Storage descriptors used during the provisioning and consumption
+        of Persistent Volumes.
+      displayName: Storage Configuration
+    - path: storageConfig.cassandraDataVolumeClaimSpec.storageClassName
+      description: "Name of the storage class where data will reside. This must be \ndefined within the Kubernetes prior to provisioning a datacenter.\n"
+      displayName: Storage Class
+      x-descriptors:
+        - urn:alm:descriptor:io.kubernetes:StorageClass
+    - path: storageConfig.cassandraDataVolumeClaimSpec.accessModes[0]
+      description: |
+        Access mode of the volume
+      displayName: Access Modes
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:select:ReadWriteOnce
+    - path: storageConfig.cassandraDataVolumeClaimSpec.resources.requests.storage
+      description: |
+        Storage requirements for the Cassandra data volume.
+      displayName: Data volume size
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.number
+    - path: storageConfig.additionalVolumes
+      description: |
+        Collection of additional storage volumes
+      displayName: Additional Volumes
+    - path: storageConfig.additionalVolumes[0].name
+      displayName: Name
+      description: |
+        Volume name
+      x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:text'
+    - path: storageConfig.additionalVolumes[0].accessModes
+      displayName: Access Modes
+      description: |
+        Volume access modes
+      x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:text'
+    - path: storageConfig.additionalVolumes[0].resources.requests.storage
+      description: |
+        Storage requirements for the additional data volume.
+      displayName: Data volume size
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.number
```

Same as the Cassandra data volume, this should be a text field.

```suggestion
    - path: storageConfig.additionalVolumes[0].resources.requests.storage
      description: |
        Storage requirements for the additional data volume. Ex: 1 Gi
      displayName: Data volume size
      x-descriptors:
        - urn:alm:descriptor:com.tectonic.text
```